uber/models.py

```diff
@@ -1000,18 +969,17 @@ class Attendee(MagModel, TakesPaymentMixin):
 
     badge_printed_name = Column(UnicodeText)
 
-    staffing         = Column(Boolean, default=False)
-    fire_safety_cert = Column(UnicodeText)
-    requested_depts  = Column(MultiChoice(c.JOB_INTEREST_OPTS))
-    assigned_depts   = Column(MultiChoice(c.JOB_LOCATION_OPTS), admin_only=True)
-    trusted          = Column(Boolean, default=False, admin_only=True)
-    nonshift_hours   = Column(Integer, default=0, admin_only=True)
-    past_years       = Column(UnicodeText, admin_only=True)
+    staffing          = Column(Boolean, default=False)
+    requested_depts   = Column(MultiChoice(c.JOB_INTEREST_OPTS))
+    assigned_depts    = Column(MultiChoice(c.JOB_LOCATION_OPTS), admin_only=True)
+    trusted           = Column(Boolean, default=False, admin_only=True)
+    nonshift_hours    = Column(Integer, default=0, admin_only=True)
+    past_years        = Column(UnicodeText, admin_only=True)
+    can_work_setup    = Column(Boolean, default=False, admin_only=True)
+    can_work_teardown = Column(Boolean, default=False, admin_only=True)
```

Comment:
``can_work_setup`` and ``can_work_teardown`` are both new.  Previously we determined this by looking at whether someone was approved for setup/teardown nights.  This still happens by default in the other plugin in a ``@presave_adjustment``, so there won't really be any difference, except that we can theoretically now set it for someone who isn't getting hotel space.